addon/components/o-s-s/input-container.ts

```diff
@@ -1,9 +1,16 @@
 import { action } from '@ember/object';
 import Component from '@glimmer/component';
 
+type FeedbackMessageType = 'error' | 'warning' | 'success';
+type FeedbackMessage = {
+  type: FeedbackMessageType;
```

Comment:
as there are not a lot of types, I think the types from `FeedbackMessageType` can directly be defined on this line.
You should also `export FeedbackMessage` so that we can use it from parent components 🙂